### PR TITLE
Added jpswalsh/academicons package

### PR DESCRIPTION
**Pull request by davidenegrini**
~~Pull request for issue: #~~
~~Related issue(s): # #~~

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Profile of the lib
 * Git repository (required): https://github.com/jpswalsh/academicons.git
 * Official website (optional, not the repository): http://jpswalsh.github.io/academicons/
 * License and its reference: [SIL OFL 1.1](http://scripts.sil.org/OFL), [MIT](http://opensource.org/licenses/mit-license.html)
 * GitHub popularity (required):
   - Count of watchers: 20
   - Count of stars: 144
   - Count of forks: 16

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 Stars on GitHub
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit (not needed)
 * [x] Close corresponding issue in commit message (no issues)
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/academicons/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "univeristy"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jpswalsh/academicons.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/*",
```
Comment (kennynaoh) [inline ajax/libs/academicons/package.json]: Hello @davidenegrini I think `css/academicons*` is better

File: ajax/libs/academicons/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "academicons",
+  "filename": "css/academicons.min.css",
+  "version": "1.7.0",
+  "description": "Academicons is an icon font for academics. It aims to supplement other packages (such as Font Awesome) with icons for academic websites.",
+  "homepage": "http://jpswalsh.github.io/academicons/",
```
Comment (kennynaoh) [inline ajax/libs/academicons/package.json]: Let's use https here.

**Comment by kennynaoh** [comment]
Please upload the files which you want to add. Thanks

**Comment by davidenegrini** [comment]
Thank you @kennynaoh for the fast revision, I'll work on it asap! I copied without big changes the font-awesome package.json in which you can find the same errors.

**Comment by PeterDaveHello** [comment]
> Please upload the files which you want to add. Thanks

We can add library with single package.json, no need to update the assets.

**Pull request closed without merging**